routes/routes.js

```diff
@@ -0,0 +1,26 @@
+const passport = require('passport');
```

Comment:
here you're using a different `passport` instance then the one you configured in `config/passport`

Reply:
I replaced it and I get "TypeError: passport.use is not a function"